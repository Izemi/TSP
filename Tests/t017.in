BDL,41.93916667,-72.68333333
BNA,36.12444444,-86.67805556
BWI,39.17527778,-76.66833333
CLE,41.40944444,-81.855
CMH,39.99694444,-82.89222222
IND,39.71722222,-86.29472222
PIT,40.49138889,-80.23277778
SYR,43.11111111,-76.10638889
SEA,47.45,-122.3116667
PAE,47.90694444,-122.2816667
