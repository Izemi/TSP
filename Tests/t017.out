-given    :       805.78 HVN ALB MHT BDL ORH PVD HVN
-insert   :       721.47 HVN BDL ALB ORH MHT PVD HVN
